packages/flutter_tools/templates/create/.analysis_options.tmpl

```diff
@@ -0,0 +1,70 @@
+# This file contains the analysis options used by the Dart analyzer
+# to decide what errors, warnings, and hints should be shown.
+# See the configuration guide for more
+# https://github.com/dart-lang/sdk/tree/master/pkg/analyzer#configuring-the-analyzer
+
+analyzer:
+  language:
+    enableStrictCallChecks: true
+    enableSuperMixins: true
+  strong-mode: false
```

Comment:
I feel like you're confusing the settings that the Flutter project wishes to apply to its own code with the settings that we recommend for our clients.  Those are very different things with different constraints.